superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/plugin/transformProps.ts

```diff
@@ -261,12 +271,22 @@ export default function transformProps(chartProps: CccsGridChartProps) {
         const isSortable = true;
         const enableRowGroup = true;
         const columnDescription = columnDescriptionMap[column];
+        const rowGroupIndex = default_group_by.findIndex((element: any) => { 
+          return element === column
+        });
+        const initialRowGroupIndex = rowGroupIndex;
+        const rowGroup = rowGroupIndex >= 0
```

Comment:
I would add brackets for readability here
```suggestion
        const rowGroup = (rowGroupIndex >= 0)
```